docs/preproduction/vision-document.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: The Vision Document
+nav_order: 1
+description: "The Vision Document Page"
+permalink: /the-vision-document
+parent: Preproduction
+---
+
+# The Vision Document
+
+The vision document (sometimes called the pitch document) is essentially a condensed and more to-the-point version of a game design document. It's exactly what it sounds like, it serves to clarify the *vision* of your game to the reader so that they are able to understand the most important aspects. This document can be an important part of getting people to invest in your game that goes hand in hand with [the pitch]("../pitch.md"), whether it be with money, time, or valuable insight.
+
+## Document Structure
+
+The style and structure you choose to outline your document with matters less than the actual information, because format tends to vary from one developer/publisher to the next. Below are some examples of vision documents and how you may want to organize them:
+
+[Example 1]("../assets/vision-docs/on_design_documents.pdf") | [Example 2]() | [Example 3]()
+
+## The Information to Outline
+
+To start, I'm going to walk through some of the information you might want to include in this document that is most important to know about your game. 
+
+### The Key Info 
+
+At the very top of the vision document, you will want to include some bullet points or a short section with the most relevent information. 
+
+* **Game Name**
+* **Genre** - 3D Action/FPS? 2D Puzzle platformer?
+* **Platform** - iOS, PC, WebGL, etc
+* **Target Audience** - age range or specific demographic?
+* **Engine** - we are using Unity Engine
+* **Controls/Input** - WASD, touch screen, etc
```

Comment:
I think this basic info is conveyed in the platform info, but it's good to include a control mapping, that helps people visualize more about the game.